### ISSUE TITLE
[QA] Option 'Scan external storages' cannot be enabled in the web UI once disabled

### ISSUE THREAD

**Issue by jnweiger** [owncloud/search_elastic#260]
Seen with search_elastic 2.1.0 RC1 (and 2.0.0 -> https://github.com/owncloud/search_elastic/issues/256)

 *   admin enables app search_elastic
 *   admin navigates to the search settings. The option 'Scan external storages' is enabled by default.
 *   admin clicks the green checkbox to **disable** the option, and clicks 'Save Settings'.
 *   After a refresh the option stays disabled. OK
 *   The database now has a scanExternalStorages entry, that was not there before:
   `mysql owncloud -e 'select * from oc_appconfig where appid = "search_elastic";'`
```
+----------------+----------------------+------------------------------------+
| appid          | configkey            | configvalue                        |
+----------------+----------------------+------------------------------------+
| search_elastic | enabled              | yes                                |
| search_elastic | installed_version    | 2.1.0                              |
| search_elastic | scanExternalStorages |                                    |
| search_elastic | servers              | elastic:ua9116rq7e@172.17.0.3:9200 |
| search_elastic | types                | filesystem                         |
+----------------+----------------------+------------------------------------+
```
* The empty string is apparently used to represent the disabled state. Okayish
* admin **enables** the option, saves and refreshes the web UI. It is disabled. BAD.
* The database now has
```
+----------------+----------------------+------------------------------------+
| appid          | configkey            | configvalue                        |
+----------------+----------------------+------------------------------------+
| search_elastic | enabled              | yes                                |
| search_elastic | installed_version    | 2.1.0                              |
| search_elastic | scanExternalStorages | 1                                  |
| search_elastic | servers              | elastic:ua9116rq7e@172.17.0.3:9200 |
| search_elastic | types                | filesystem                         |
+----------------+----------------------+------------------------------------+
```
* admin tries `occ config:app:set search_elastic scanExternalStorages  --value true`
   and several other plausible values. Nothing enables the option. BAD

#### Expected behaviour:
* Setting the checkbox enables the option.
* A value of "1" (or "yes" or "true") would enable the option.

#### Workaround:
* `occ config:app:delete search_elastic scanExternalStorages`
  -> Now the setting is **enabled** :upside_down_face:

**Comment by pmaier1** [comment]
Blocker for 2.1.0 release. Please check @ahherrera.

**Comment by jnweiger** [comment]
Re-Trying with multiple browser sessions, and incognito mode, and browser cache cleanup.

I cannot find a away via gui to change the value of ```
occ config:app:get search_elastic scanExternalStorages
1
```
back to something else from the GUI. The GUI always refresehes with an unset checkbox, 
regardless, If I click that once or multiple times, the value in the db remains at `1` which is not in sync with what is displayed.

**Comment by jnweiger** [comment]
Confirmed fixed in 2.1.0-rc3
